index.html

```diff
@@ -183,40 +188,75 @@ <h3>LET'S CONNECT</h3>
           <button type="button">Get my resume</button>
         </div>
         <div class="Language">
-          <div class="lang">
+          <div class="lang flex-row">
             <h2>Languages</h2>
             <i class="fa fa-angle-down fa-2x" aria-hidden="true"></i>
           </div>
           <div class="Lang-box">
             <div class="js-img-box">
-              <img src="img/Ellipse 1.png" alt="" />
               <ul>
-                <li>Javascript</li>
+                <li><img src="img/Ellipse 1.png" alt="" /></li>
+                <li class="L1">Javascript</li>
               </ul>
             </div>
             <div class="html-img-box">
-              <img src="img/Ellipse 1 (1).png" alt="" />
               <ul>
-                <li>HTML</li>
+                <li><img src="img/Ellipse 1 (1).png" alt="" /></li>
+                <li class="L2">HTML</li>
               </ul>
             </div>
             <div class="css-img-box">
-              <img src="img/Ellipse 1 (2).png" alt="" />
               <ul>
-                <li>CSS</li>
+                <li><img src="img/Ellipse 1 (2).png" alt="" /></li>
+                <li class="L3">CSS</li>
               </ul>
             </div>
           </div>
-          <div class="Frame">
+          <div class="Frame flex-row">
             <p>Frameworks</p>
             <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
           </div>
-          <div class="skill">
+          <div class="skill flex-row">
             <p>Skills</p>
             <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
           </div>
         </div>
       </section>
+      <section class="contact-form flex-column">
+        <h3>Contact me</h3>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      <form action="https://formspree.io/f/mayzpyer" method="post">
+        <input
+          class="form"
+          type="text"
+          name="name"
+          placeholder="Name"
+          maxlength="30"
+          required
+        />
+        <input
+          class="form"
+          type="email"
+          name="email"
+          placeholder="Email"
+          maxlength="30"
+          required
+        />
+        <textarea
+          name="message"
+          class="textarea"
+          placeholder="Enter your message here..."
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+        ></textarea>
```

Comment:
Figma (input fields and the text area have the same width)
![image](https://user-images.githubusercontent.com/78251927/228901698-847a9f73-b5b9-46b3-8f90-06bdbcbefa01.png)

Your design (input fields have smaller width compared to the textarea)
![image](https://user-images.githubusercontent.com/78251927/228900728-0296286f-5ccc-4a7a-ac67-2ecd380fba6b.png)

Please adjust this section to ensure that `input` fields and the `textarea` have equal `width` as given in the Figma design. 🤗